src/utils/notification/htmlBuilder.ts

```diff
@@ -329,6 +333,15 @@ export const buildTable = (
         ) {
           table += `<td  style = "color: #000; font-size: 15px; font-family: 'Roboto', Arial, sans-serif; padding-left: 20px; padding-top: 8px;padding-bottom: 8px; border-bottom:1px solid #d1d5db;">
           ${formatDates(_.get(record, field.name))}</td>`;
+        } else if (field.type === 'geospatial') {
+          table += `<td  style = "color: #000; font-size: 15px; font-family: 'Roboto', Arial, sans-serif; padding-left: 20px; padding-top: 8px;padding-bottom: 8px; border-bottom:1px solid #d1d5db;">
+          ${formatDates(
```

Comment:
@HasnatL-Adapptlimited 
I'll merge it like that, but formatDates is not correct method call, as it's obviously not a date
It would be better to call something like "format"

Reply:
@HasnatL-Adapptlimited @morgang-adappt 

seems like I can't push on your repo, so I can't push my change, and I don't want to create another branch just for that

the solution would be to use lodash like that:

get(record.data, `${field.name}.properties.countryName`)

and the same for the 2 other ones